### PR TITLE
Feature/comments on articles

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/UserWidget/UserWidgetSmall.re [hunk 1]
```diff
@@ -10,7 +10,7 @@ module Styles = {
         height(px(30)),
         width(px(30)),
         borderRadius(px(15)),
-        marginRight(px(8)),
+        marginRight(px(10)),
```
Comment (davodesign84) [inline packages/kauri-components/components/UserWidget/UserWidgetSmall.re]: Maybe be worth porting this back to JS and use the theme?

File: packages/kauri-web/components/containers/SubmitArticleForm/View.js [hunk 2]
```diff
@@ -125,17 +126,19 @@ class SubmitArticleForm extends React.Component<Props> {
...
             // }
             if (typeof article_id === 'string' && submissionType === 'submit/update') {
-              const { id, version, status }: ArticleDTO = this.props.data && this.props.data.getArticle;
+              const { id, version, status, author }: ArticleDTO = this.props.data && this.props.data.getArticle;
 
               if (status === 'PUBLISHED') {
                 // Here I am really submitting a new article with updates for an already existing article!
                 // Not my published article, I clicked Update article version, I create a new article and self publish it AIO
-                return submitArticleAction({
+                console.log(author.id)
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/SubmitArticleForm/View.js]: console.log

File: packages/kauri-components/components/UserWidget/UserWidgetSmall.re (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/UserWidget/UserWidgetSmall.re]: Will fix in another PR if that's cool

File: packages/kauri-components/components/Typography/index.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+  font-weight: normal;
+  letter-spacing: -0.1px;
+  line-height: 18px;
+`
+
+type TypographyType = {
+  name?: string, // START NAME WITH A CAPITAL LETTER NOOB AND NO SPACES
+  as?: string,
+  fontSize?: 11 | 13 | 14 | 16 | 18 | 20 | 22 | 28,
+  fontWeight?: 300 | 'normal' | 500 | 'bold',
```
[the remark this answers is not in the document]
Comment (rej156) [reply packages/kauri-components/components/Typography/index.js]: that is four 📦

[the remark this answers is not in the document]
Comment (rej156) [reply packages/kauri-components/components/Typography/index.js]: idk took them out figma

File: packages/kauri-components/components/Typography/index.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+  textTransform?: 'uppercase' | 'lowercase' | 'capitalize',
+  color?: string,
+  hoverColor?: string,
+  component?: ReactComponentClass<{}>
+}
+
+const typographyTypes: Array<TypographyType> = [
```
[the remark this answers is not in the document]
Comment (rej156) [reply packages/kauri-components/components/Typography/index.js]: will amend

**Pull request merged**